assignments/array-methods.js

```diff
@@ -55,22 +55,37 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+let fullName = []
+
+runners.forEach(function (value) {
+   fullName.push(value.first_name + " " + value.last_name)
```

Comment:
This is a fine way to construct strings. Remember that you may see the template literal syntax as well:

```js
`${value.first_name} ${value.last_name}`;
```